wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/ElevatorSim.java

```diff
@@ -67,29 +67,29 @@ public ElevatorSim(
    * @param kV The velocity gain.
    * @param kA The acceleration gain.
    * @param gearbox The type of and number of motors in the elevator gearbox.
-   * @param minHeightMeters The min allowable height of the elevator.
-   * @param maxHeightMeters The max allowable height of the elevator.
+   * @param minHeight The min allowable height of the elevator.
+   * @param maxHeight The max allowable height of the elevator.
    * @param simulateGravity Whether gravity should be simulated or not.
-   * @param startingHeightMeters The starting height of the elevator.
+   * @param startingHeight The starting height of the elevator.
    * @param measurementStdDevs The standard deviations of the measurements. Can be omitted if no
    *     noise is desired. If present must have 1 element for position.
    */
   public ElevatorSim(
```

Comment:
Units are missing in the `@param` comments for `minHeight`, `maxHeight`, and `startingHeight`.